src/Components/ArticleCard/ArticleCard.module.scss

```diff
@@ -0,0 +1,57 @@
+.main {
+  font-family: var(--gdsc-font-1);
+  color: var(--gdsc-grayish-1-100);
+  width: min(90%, 50rem);
+  margin: 2rem auto;
+
+  .heading {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .title {
+    font-size: 1.5rem;
+    font-weight: var(--heading-1-text-weight);
+    cursor: pointer;
+  }
+
+  .title:hover {
+    color: var(--gdsc-blue-1-100);
+  }
+
+  .subheading {
+    font-weight: var(--paragraph-1-text-weight);
+    font-size: 0.8rem;
+    text-align: justify;
+
+    /* Donot exceed more than 5 lines of content in Articles Page */
```

Comment:
Specify exactly how many characters will properly fit in the article card. Truncate the rest.